Add docs for using the Kubernetes monitor on self-hosted instances (#2755)
OctopusDeploy/docs: src/pages/docs/installation/octopus-server-linux-container/octopus-in-kubernetes.mdx

Unlike the Octopus Web Portal, Polling Tentacles must be able to connect to each Octopus node individually to pick up new tasks. Our Octopus HA cluster assumes two nodes, therefore a load balancer is required for each node to allow direct access.

> The following YAML creates load balancers with separate public IPs for each node. They direct web traffic to each node on port `80`, Polling Tentacle traffic on port `10943` and gRPC traffic on port `8443`.

Reviewer: Nit: I'm an oxford comma stan - feel free to discard if it's against our style guides.
```suggestion
The following YAML creates load balancers with separate public IPs for each node. They direct web traffic to each node on port `80`, Polling Tentacle traffic on port `10943`, and gRPC traffic on port `8443`.
```

Reviewer: I knew I was on the right side of the fence: https://www.octopus.design/latest/brand/writing/grammar-rules-8WpUimhK-8WpUimhK#section-the-oxford-comma-7b

To find out more about storage classes, refer to the [Kubernetes Storage Classes](https://kubernetes.io/docs/concepts/storage/storage-classes/) documentation.
:::

> While it is possible to mount external storage by manually defining Persistent Volume definitions in YAML, Cloud providers offering Kubernetes managed services typically include the option to dynamically provision file storage based on persistent volume claim definitions. 

Reviewer: Nit: Consider changing to be less formal, since you're also changing from whilst.
```suggestion
While it's possible to mount external storage by manually defining Persistent Volume definitions in YAML, Cloud providers offering Kubernetes managed services typically include the option to dynamically provision file storage based on persistent volume claim definitions. 
```
